packages/jackett/build.sh

```diff
@@ -0,0 +1,43 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/jackett/jackett"
+TERMUX_PKG_DESCRIPTION="API Support for your favorite torrent trackers"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="0.22.1468"
+TERMUX_PKG_SRCURL="https://github.com/Jackett/Jackett/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz"
+TERMUX_PKG_SHA256="19a3017691273a6505a2a64046e5459b97e080e7469d9cab2f246bc04e62a40e"
+TERMUX_PKG_BUILD_DEPENDS="aspnetcore-targeting-pack-8.0, dotnet-targeting-pack-8.0"
+TERMUX_PKG_DEPENDS="aspnetcore-runtime-8.0, dotnet-host, dotnet-runtime-8.0"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_SERVICE_SCRIPT=("jackett" "exec ${TERMUX_PREFIX}/bin/jackett --DataFolder ${TERMUX_ANDROID_HOME}/.config/jackett 2>&1")
+TERMUX_PKG_BLACKLISTED_ARCHES="arm"
+TERMUX_PKG_RM_AFTER_INSTALL="
+lib/jackett/libMono.Unix.so
+"
+
+termux_step_pre_configure() {
+	export DOTNET_CLI_TELEMETRY_OPTOUT=1
+	termux_setup_dotnet
+}
+
+termux_step_make() {
+	dotnet publish src/Jackett.Server \
+	--framework "net8.0" \
+	--no-self-contained \
+	--runtime "$DOTNET_TARGET_NAME" \
+	--configuration Release \
+	--output build/ \
+	/p:AssemblyVersion="${TERMUX_PKG_VERSION}" \
+	/p:FileVersion="${TERMUX_PKG_VERSION}" \
+	/p:InformationalVersion="${TERMUX_PKG_VERSION}" \
+	/p:Version="${TERMUX_PKG_VERSION}"
+	dotnet build-server shutdown
+}
+
+termux_step_make_install() {
+	cp -r build "${TERMUX_PREFIX}/lib/jackett"
```

Comment:
This probably shouldn't be a full recursive copy.
Consider using `TERMUX_PKG_RM_AFTER_INSTALL`[^1] to specifically get rid of the files we do not need.

For example the `.pdb` program debug files, `jackett.pdb` `Jackett.Common.pdb` and `DateTimeRoutines.pdb` as well as the `README.md` and `LICENSE` file, which is handled by the `termux_step_license_install` function in the build system.

[^1]: https://github.com/termux/termux-packages/wiki/Creating-new-package#:~:text=TERMUX_PKG_RM_AFTER_INSTALL

Reply:
You can do space or newline separated for `TERMUX_PKG_RM_AFTER_INSTALL`.
https://github.com/termux/termux-packages/blob/054de3f8fe10c4f51a1a518142fd0981099c704f/packages/groff/build.sh#L15-L19

You shouldn't need to overwrite `termux_step_make_install()` at all.
You can just move creating the `bin/jackett` shim to `termux_step_post_make_install()`, which is where that is done for `neovim` for example.
https://github.com/termux/termux-packages/blob/054de3f8fe10c4f51a1a518142fd0981099c704f/packages/neovim/build.sh#L79-L92
(Although in that case the shim was spun out into a separate file instead of a heredoc. Which doesn't seem necessary here.)

Reply:
I can take a look at it in a bit.